Source/SmallBasic.Editor/Components/Pages/Debug/MemoryExplorer.scss

```diff
@@ -295,8 +294,9 @@ memory-explorer {
                     padding: 10px;
                     display: flex;
                     flex-direction: column;
-                    justify-content: flex-end;
+                    justify-content: flex-start;
```

Comment:
I tested your changes, and found out that only these two properties on `container` are needed:
```css
                    justify-content: flex-start;
                    overflow-y: scroll;
```
Can you please explain the rest?

Reply:
https://github.com/sb/smallbasic-editor/blob/a3b2823c20c02628f2e22e32483baa3b24308430/Source/SmallBasic.Editor/Libraries/Graphics/ImageGraphicsObject.cs#L23-L31

At these lines, we are checking if the string is a valid http/https link so that users can't access certain files. But the way, we handle it in the event of a bad request was to leave the String as an empty String.

https://github.com/sb/smallbasic-editor/blob/a3b2823c20c02628f2e22e32483baa3b24308430/Source/SmallBasic.Editor/Libraries/Graphics/ImageGraphicsObject.cs#L45-L53

At these lines, changing the tag to be an img file renders it.